Assets/Scripts/MainGame/ShopSystem/ShopManager.cs

```diff
@@ -26,7 +26,7 @@ void Start()
 
     public void RefreshCredits() 
     {
-        credit = ContentDistributor.contentDistributor.mainButton.credits; //dummy
+        credit = Account.credits; //dummy
```

Comment:
//dummy könnte man auch entfernen, ist ja jetzt die richtige Implementierung.

Reply:
Das Script "DummyButton" hat eine Methode MainButtonAction, die meiner Meinung nach nicht zum Script Account gehört

<img width="529" alt="Screenshot 2022-11-27 at 19 12 03" src="https://user-images.githubusercontent.com/59295494/204152437-c24e3b82-ae99-451d-b229-c85853c3d9bc.png">

Reply:
Unnötige Methode aus Dummy sind entfernt

Reply:
//dummy comment is deleted